tensorboard/webapp/metrics/views/right_pane/scalar_column_editor/scalar_column_editor_container.ts

```diff
@@ -48,16 +56,27 @@ import {DataTableMode} from '../../../../widgets/data_table/types';
 export class ScalarColumnEditorContainer {
   constructor(private readonly store: Store<State>) {}
 
-  readonly singleHeaders$ = this.store.select(getSingleSelectionHeaders);
-  readonly rangeHeaders$ = this.store.select(getRangeSelectionHeaders);
+  readonly singleHeaders$ = this.store
```

Comment:
runs are now hidden in the edit menu so they will no longer be sortable or hidable